CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `**kwargs` argument for passing additional arguments to base class ([#833](https://github.com/PyTorchLightning/metrics/pull/833))
 
 
+- Added argument `store_on_cpu` to `FrechetInceptionDistance`, `KernelInceptionDistance`, `InceptionScore` to store features on cpu ([#851](https://github.com/PyTorchLightning/metrics/pull/851))
```

Comment:
I think that this could be a very similar case for all metrics using a DNN under the hood some for example other Text metrics...

Reply:
Not necessarily all such metrics. For example the `lpips` also uses a DNN underneat but only need to store a scalar value:
https://github.com/PyTorchLightning/metrics/blob/b0bcccb12ee80cbf72e596d3bf43dfb533e30e22/torchmetrics/image/lpip.py#L140-L142
Instead it would be appropriate for all metrics where the state can be large